views/designerHTML.ejs

```diff
@@ -1,8 +1,14 @@
+<div class="deprecation" id="deprecation">
+  <div class="closeWarning" id="closeWarning" onclick="this.parentNode.remove()">X</div>
+    Mozilla's tools for teaching the Web are evolving. Beginning June 19, you will not be able to create or edit Appmaker projects. To learn more, read <a href="http://mzl.la/1ACom34" target="_blank">our blog post.</a>
```

Comment:
This needs to be transformed by i18n. Check the rest of the `ejs` code for some examples about how to do that.